### PR TITLE
[PATCH v2] linux-gen: dpdk: port ID is two bytes long

### PR DESCRIPTION
**Pull request by matiaselo**
V2
- Added alignment directive to pkt_dpdk_t struct (Dmitry)

### CODE REVIEW

File: platform/linux-generic/include/odp_packet_dpdk.h [hunk 1]
```diff
@@ -54,12 +54,12 @@ typedef struct ODP_ALIGNED_CACHE {
...
 	odp_pktin_hash_proto_t hash;	  /**< Packet input hash protocol */
 	uint16_t mtu;			  /**< maximum transmission unit */
+	uint16_t port_id;		  /**< DPDK port identifier */
 	/** Use system call to get/set vdev promisc mode */
 	uint8_t vdev_sysc_promisc;
 	uint8_t lockless_rx;		  /**< no locking for rx */
 	uint8_t lockless_tx;		  /**< no locking for tx */
-	uint8_t port_id;			  /**< DPDK port identifier */
-	/* --- 34 bytes --- */
+	/* --- 35 bytes --- */
```
Comment (Bill-Fischofer-Linaro) [inline platform/linux-generic/include/odp_packet_dpdk.h]: Perhaps add an explicit 5 byte pad field here?

Comment (lumag) [reply platform/linux-generic/include/odp_packet_dpdk.h]: Or just alignment directive

Comment (matiaselo) [reply platform/linux-generic/include/odp_packet_dpdk.h]: Alignment directive makes sense, will fix.

**Comment by muvarov** [comment]
Merged.

**Pull request closed without merging**